lib/src/features/login/presentation/login_screen.dart

```diff
@@ -112,9 +112,8 @@ class _LoginScreenState extends ConsumerState<LoginScreen> {
                   children: [
                     Padding(
                       padding: const EdgeInsets.all(8.0),
-                      child: AlcanciaToolbar(
-                        state: StateToolbar.logoNoletters,
-                        logoHeight: size.height / 8,
+                      child: AlcanciaLogo(
```

Comment:
le puedes quitar la coma? en este caso, creo que se ve mejor que este en una sóla línea
```suggestion
child: AlcanciaLogo(height: size.height / 8),
```